open-machine-learning-jupyter-book/assignments/python-programming-basics.ipynb

```diff
@@ -1,59 +1,215 @@
-{

```

Comment:
<p>We only have uppercase for the first letter in title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>